etherman/etherman.go

```diff
@@ -46,6 +46,8 @@ var (
 
 	// ErrNotFound is used when the object is not found
 	ErrNotFound = errors.New("Not found")
+	// ErrIsReadOnlyMode is used when the EtherMan client is in read-only mode.
+	ErrIsReadOnlyMode = errors.New("etherman client in read-only mode")
```

Comment:
I was thinking about having a better error message to allow the user to understand why this is running on read-only mode.

maybe if we complement the error message with the reason, something like `etherman client in read-only mode, because <add something here that makes sense>`, wdyt?

I was thinking something like this: "etherman client in read-only mode because there was no account configured to send transactions to L1. Please check the [Etherman] PrivateKeyPath and PrivateKeyPassword configuration."